webapi/src/main/scala/org/knora/webapi/slice/infrastructure/JwtService.scala

```diff
@@ -162,13 +161,10 @@ final case class JwtServiceLive(
    * @return a [[Boolean]].
    */
   override def validateToken(token: String): Task[Boolean] =
-    ZIO.attempt(if (CacheUtil.get[User](AUTHENTICATION_INVALIDATION_CACHE_NAME, token).nonEmpty) {
-      // token invalidated so no need to decode
-      logger.debug("validateToken - token found in invalidation cache, so not valid")
-      false
-    } else {
-      decodeToken(token).isDefined
-    })
+    if (cache.contains(token)) ZIO.succeed(false)
+    else {
+      ZIO.succeed(decodeToken(token).isDefined)
+    }
```

Comment:
Convert to `ZIO.succeed(!contains && isDefined)`?

Reply:
Makes sense: https://github.com/dasch-swiss/dsp-api/pull/3201/commits/722e6612dd781e834ad591da73c830b90b1af666